client/watcher/watcher.go

```diff
@@ -20,6 +20,9 @@ type IFace interface {
 	ReadFile() ([]byte, error)
 }
 
+// watchWaitTime is the maximum time to wait before reloading changes
+const watchWaitTime = 5 * time.Second
+
 // Watcher is a wrapper for `fsnotify` that provides the
 // registration of a callback for WRITE events.
```

Comment:
maybe add to the Godoc here indicating that `Watcher` does polling as a fallback.